routes/feedback/feedback.controller.js

```diff
@@ -0,0 +1,60 @@
+/* istanbul ignore file */
+module.exports = (app, route) => {
+  route.draw(app)
+    .post((req, res) => {
+      const date = new Date();
+
+      const feedback = {
+        "problems": req.body.problems || "n/a",
+        "session": req.locals.session.id || "n/a",
+        "version": process.env.GITHUB_SHA || "n/a",
+        "url": req.headers.referer || req.headers.referrer || "n/a",
+        "date": date.toISOString(),
+        "language": res.locals.getLocale(),
+      }
+      console.log(JSON.stringify({ "feedback": feedback }));
+
+      sendNotification(feedback);
+      saveToAirtable(feedback);
+
+      return res.redirect(res.locals.routePath('feedback-thanks'));
+    })
+}
+
+const sendNotification = (feedback) => {
+  if (!(process.env.NOTIFY_API_KEY && process.env.FEEDBACK_EMAIL_TO && process.env.NOTIFY_ENDPOINT)) {
```

Comment:
I feel like these should be ORs.

Reply:
All three need to exist for this to work ... I know it's a bit odd to read with the ! operator before the sub-expression. Basically, just bailing out if any of the env vars are missing.